### ISSUE TITLE
Something wrong

### ISSUE THREAD

**Issue by corkiyao** [RujingD/Maxim_pytorch#1]
I always have the question:  when iteration over 100 iteration, loss is nan. Is my initialization wrong? Well, I use:         self.apply(self._init_weights)
    def _init_weights(self, m):
        if isinstance(m, nn.Linear):
            trunc_normal_(m.weight, std=.02)
            if isinstance(m, nn.Linear) and m.bias is not None:
                nn.init.constant_(m.bias, 0)  
Can you give me some advice? Thank you
